src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,3 +1,37 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => <>Put the card here</>;
+export const Card = ({ movie }) => {
```

Comment:
It's not just card, use more descriptive name
```suggestion
export const MovieCard = ({ movie }) => {
```